sql/popularize.go

```diff
@@ -0,0 +1,50 @@
+package sql
```

Comment:
I see how comes the content of this file. I would recommend to move it from `sql/popularize.go` into `sql/testdata/popularize.go`.

Reply:
Comments done, thanks for advices.